pages/common/gum.md

```diff
@@ -0,0 +1,28 @@
+# gum
+
+> A tool for making glamorous shell scripts.
+> More information: <https://github.com/charmbracelet/gum>.
+
+- Interactively pick an option to print to `STDOUT`:
```

Comment:
```suggestion
- Interactively pick a specific option to print to `STDOUT`:
```

Reply:
Is STDOUT in caps necessary?

Reply:
I like this but our style guide doesn't provide any info about how we have to write stdin, stdout, stderr.

Reply:
Looking at what we currently use, lowercase is more used.